app/views/merchant/items/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= link_to @merchant.name, "/merchants/#{@merchant.id}"%><span> Items</span></h1>
+<p align="center"><%= link_to "Add New Item", "/merchants/#{@merchant.id}/items/new" %></p>
```

Comment:
Can you take out `align="center"` and instead give it an id and style that id in the CSS stylesheet?